nltk/translate/ibm1.py

```diff
@@ -125,6 +125,18 @@ def __init__(self, sentence_aligned_corpus, iterations, probability_tables=None)
             ``translation_table``.
             See ``IBMModel`` for the type and purpose of this table.
         :type probability_tables: dict[str]: object
+
+
+        :param stepSize: Optional. Use this to mix old probability table  
```

Comment:
I am curious if you have considered making the batch_size and alpha as parameters instead of step_size?
This would make it easier to use because we can just call ibm1 without having to wrap it in while loops.